RemoteHealthcare/HRManager.cs

```diff
@@ -80,7 +71,7 @@ private static void BleHeart_SubscriptionValueChanged(object sender, BLESubscrip
             }
         }
 
-        public bool closeConnections()
+        public bool CloseConnections()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)